LibreNMS/Alert/Transport/Api.php

```diff
@@ -83,11 +104,30 @@ public static function configTemplate()
                     'name' => 'api-url',
                     'descr' => 'API URL',
                     'type' => 'text',
+                ],
+                [
+                    'title' => 'Options',
+                    'name' => 'api-options',
+                    'descr' => 'Enter the options (format: option=value separated by new lines)',
+                    'type' => 'textarea',
+                ],
+                [
+                    'title' => 'Auth Username',
+                    'name' => 'api-auth-username',
+                    'descr' => 'Auth Username',
+                    'type' => 'text',
+                ],
+                [
+                    'title' => 'Auth Password',
+                    'name' => 'api-auth-password',
+                    'descr' => 'Auth Password',
+                    'type' => 'password',
                 ]
             ],
             'validation' => [
                 'api-method' => 'in:GET,POST',
-                'api-url' => 'required|url'
+                'api-url' => 'required|url',
+                'api-options' => 'required|string'
```

Comment:
api-options probably should not be required

Reply:
Correct, I'll change that.